diesel_tests/tests/only.rs

```diff
@@ -0,0 +1,236 @@
+use crate::schema::connection;
+// use crate::schema_dsl::*;
+use diesel::dsl::*;
+use diesel::pg::expression::dsl::OnlyDsl;
+use diesel::*;
```

Comment:
I would prefer using `diesel::prelude::*;` instead.
```suggestion
use diesel::*;
```